tasks/build/package_scripts/post_install.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+user_check() {
+  getent passwd "$1" > /dev/null 2>&1
+}
+
+user_create() {
+  # Create a system user. A system user is one within the system uid range and
+  # has no expiration
+  useradd -r "$1"
+}
+
+if ! user_check "<%= user %>" ; then
+  user_create "<%= user %>"
+fi
+chown <%= user %>:<%= group %> /opt/kibana/optimize
```

Comment:
Are you sure this doesn't need to be recursive? After installing the `kibana-5.0.0_snapshot-1.i686.rpm` package, I found a bunch of folders and files under `/opt/kibana/optimize` that are owned by `root`:

```
$ tree -ug /opt/kibana/optimize | head -20
/opt/kibana/optimize
└── [root     root    ]  bundles
    ├── [root     root    ]  a573941f02f4331f81046356ebb667eb.swf
    ├── [root     root    ]  commons.bundle.js
    ├── [root     root    ]  commons.style.css
    ├── [root     root    ]  console.bundle.js
    ├── [root     root    ]  console.entry.js
    ├── [root     root    ]  console.style.css
    ├── [root     root    ]  fbe046f38bf68f1469110319dee9bbfd.js
    ├── [root     root    ]  kibana.bundle.js
    ├── [root     root    ]  kibana.entry.js
    ├── [root     root    ]  kibana.style.css
    ├── [root     root    ]  node_modules
    │   ├── [root     root    ]  bootstrap
    │   │   └── [root     root    ]  fonts
    │   │       ├── [root     root    ]  glyphicons-halflings-regular.eot
    │   │       ├── [root     root    ]  glyphicons-halflings-regular.svg
    │   │       ├── [root     root    ]  glyphicons-halflings-regular.ttf
    │   │       ├── [root     root    ]  glyphicons-halflings-regular.woff
    │   │       └── [root     root    ]  glyphicons-halflings-regular.woff2
```